tests/sentry/issues/auto_source_code_config/test_frame_info.py

```diff
@@ -35,10 +35,10 @@ def test_frame_filename_repr(self) -> None:
         path = "getsentry/billing/tax/manager.py"
         assert FrameInfo({"filename": path}).__repr__() == f"FrameInfo: {path}"
 
-    def test_raises_unsupported(self) -> None:
-        for filepath in UNSUPPORTED_FRAME_FILENAMES:
-            with pytest.raises(UnsupportedFrameInfo):
-                FrameInfo({"filename": filepath})
+    @pytest.mark.parametrize("filepath", UNSUPPORTED_FRAME_FILENAMES)
```

Comment:
Using parametrize makes it clearer which case fails. This is an orthogonal change.